src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+import auth from '../../SpoAuth';
+// import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+// import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, // CommandValidate, CommandTypes,
+  CommandTypes,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+import config from '../../../../config';
+// import { FieldLink } from './FieldLink';
+// REST URL
+// POST http://<sitecollection>/<site>/_api/web/lists(listid)/contenttypes(contenttypeidcontenttypeid})/fieldlinks(fieldlinkid)/deleteObject()
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+
+interface Options extends GlobalOptions {
```

Comment:
All remove commands should have the `confirm` option which allows to suppress the prompt for confirmation.